services/secret-service/src/route/auth-clients/index.js

```diff
@@ -257,6 +265,45 @@ class AuthClientRouter {
                 });
             }
         });
+
+        this.router.post('/:id/get-token', getKeyParameter, jsonParser, async (req, res, next) => {
```

Comment:
Is this still needed? Would it just be the same as retrieving the secret?

Reply:
Good catch